synthtool/sources/templates.py

```diff
@@ -123,6 +124,14 @@ def language_pretty(input):
     return input
 
 
+#
+# converts Foo Bar into foo-bar, for use wih anchor links.
+#
+def slugify(input: str) -> str:
+"""Converts Foo Bar into foo-bar, for use wih anchor links."""
```

Comment:
Sorry, I don't know what I was thinking. :woman_facepalming: This line should be indented.